File: testprojects/csharp/benchmark/domain/google_plus/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharpBenchmark.domain.google_plus
{
    public class Circle : Observable
    {
        private List<Contact> contacts;

        public Circle()
        {
            contacts = new List<Contact>();
        }

        public void addContact(Contact contact)
        {
            contacts.Add(contact);
        }

        public void removeContact(Contact contact)
        {
            contacts.Remove(contact);
        }
    }
}

File: testresources/java/benchmark_2012/domain/google_plus/Circle.java
package domain.google_plus;

import java.util.ArrayList;
import java.util.List;

public class Circle extends Observable {
	private List<Contact> contacts;

	public Circle(){
		contacts = new ArrayList<Contact>();
	}

	public void addContact(Contact contact){
		contacts.add(contact);
	}

	public void removeContact(Contact contact){
		contacts.remove(contact);
	}
}